src/pages/JobDescribtion/components/JobDescriptionItem.jsx

```diff
@@ -0,0 +1,45 @@
+import { useTheme } from "@emotion/react";
+import {
+  ToggleButton,
+  Grid,
+  ToggleButtonGroup,
+  ListItem,
+  ListItemText,
+} from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+function JobDescriptionItem(props) {
+  const [selected, setSelected] = React.useState(() => [0]);
+  const handleChange = (_, newSelected) => {
+    setSelected(newSelected);
+  };
+  return (
+    <Grid item xs={3}>
+      <ToggleButtonGroup value={selected} onChange={handleChange}>
+        <ToggleButton
+          value="true"
+          sx={{
+            height: "50px",
+            width: "150px",
+            background: useTheme().palette.primary.light,
+          }}
+        >
+          <ListItem sx={{ textAlign: "center", fontSize: "60px" }}>
+            <ListItemText
+              primary={props.jobDescription.skills || ""}
+            ></ListItemText>
+          </ListItem>
+        </ToggleButton>
+      </ToggleButtonGroup>
+    </Grid>
+  );
+}
+
+JobDescriptionItem.propTypes = {
+  jobDescription: PropTypes.shape({
+    skills: PropTypes.string,
```

Comment:
Why is this key called skills? If I'm not mistaken, skills is only a string and represents one skill, so skill would be better, if you don't want to replace that skill object (see comment mock data)